public_html/wp-content/plugins/wordcamp-speaker-feedback/views/form-feedback.php

```diff
@@ -110,8 +115,14 @@
 		<textarea
 			id="sft-<?php echo esc_attr( $key ); ?>"
 			name="sft-<?php echo esc_attr( $key ); ?>"
+			<?php echo $maxlength_attr; // phpcs:ignore ?>
 			<?php echo $schema[ $key ]['required'] ? 'required' : ''; ?>
 		></textarea>
+		<?php if ( $maxlength_attr ) : ?>
+			<div class="speaker-feedback__field-help">
+				0/<?php echo absint( $schema[ $key ]['attributes']['maxlength'] ); ?>
+			</div>
```

Comment:
Should this be `aria-hidden`?

Reply:
hm… no, because we want to expose that content to screen readers, but it might need more context. i'll take a look at what other char counters do for a11y.

Reply:
Added a describedby in 6cd11e9d1174bce4f706432801048c81e10b6fc9 — the counter stays available to all users (since some sighted users use screen readers), and now has more context with the extra description.